packages/query/src/block-processor.ts

```diff
@@ -36,6 +36,13 @@ import { getAssetId } from '@penumbra-zone/getters/src/metadata';
 import { STAKING_TOKEN_METADATA } from '@penumbra-zone/constants/src/assets';
 import { toDecimalExchangeRate } from '@penumbra-zone/types/src/amount';
 import { ValidatorInfoResponse } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/stake/v1/stake_pb';
+import { uint8ArrayToHex } from '@penumbra-zone/types/src/hex';
+
+declare global {
+  // `var` required for global declaration (as let/const are block scoped)
+  // eslint-disable-next-line no-var
+  var ASSERT_ROOT_VALID: boolean | undefined;
+}
```

Comment:
@grod220 can we get some docs in the dev docs about the existence of this global and how you would set it and why?

Reply:
Sounds good 👍 
https://github.com/penumbra-zone/web/pull/891